app/src/main/res/layout/fragment_wait_times.xml

```diff
@@ -3,15 +3,14 @@
 <RelativeLayout android:id="@+id/wait_time_graph"
```

Comment:
could just be naming, but i really don't think that this is a fragment -- there should be any lifecycle methods attached to this class. it's just a view delegate right?

Reply:
Addressed in latest commit - turned it into inflating a view into a holder.